src/mesh/PhoneAPI.cpp

```diff
@@ -105,8 +105,10 @@ bool PhoneAPI::handleToRadio(const uint8_t *buf, size_t bufLength)
             break;
         case meshtastic_ToRadio_mqttClientProxyMessage_tag:
             LOG_INFO("Got MqttClientProxy message\n");
-            if (mqtt && moduleConfig.mqtt.proxy_to_client_enabled) {
+            if (mqtt && moduleConfig.mqtt.proxy_to_client_enabled && moduleConfig.mqtt.enabled && channels.anyMqttEnabled()) {
```

Comment:
@GUVWAF this guard probably needs to consider the new map report enabled too right?